roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -74,6 +74,9 @@ kubernetesMasterConfig:
   masterCount: 1
   masterIP: ""
   podEvictionTimeout: ""
+  proxyClientInfo:
```

Comment:
@abutcher The easiest way will be to conditionally run openshift_facts to set the version if the state of the installation task is successful.